File: Data Structures & Algorithms/n-th-tribonacci-number/submission-1.java
class Solution {
    public int tribonacci(int n) {
         if(n==0){
            return 0;
        }else if(n==0||n==1){
            return 1;
        }
        int ft = 0;
        int st = 1;
        int tt = 1;

        for(int i = 1 ; i <= n ; i++){
            int fr = ft+st+tt;
            ft = st ; 
            st = tt;
            tt = fr; 
        }
        return ft;
    }
}

File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        // sort(nums.begin() , nums.end());
        int n = nums.size();

        for(int num : nums){
            int count = 0;
            for(int i : nums){
                if(i == num) count++;
            }
            if(count > n/2) return num;
        }
        return -1;
    }
};